qt_binder/binder.py

```diff
@@ -568,6 +564,100 @@ def _on_trait_change(self, handler, name=None, remove=False,
                 for func, name in connectors:
                     func(self, name)
 
+    def _collect_renamings(self, binder_class):
```

Comment:
this method is simply lines 581-586 from master refactored into a method - no new logic has been added here.